client/landing/jetpack-cloud/sections/backups/rewind-flow/restore.tsx

```diff
@@ -1,15 +1,215 @@
+			<div className="rewind-flow__header">
+				<Gridicon icon="history" size={ 48 } />
+			</div>
+			<h3 className="rewind-flow__title">{ translate( 'Restore site' ) }</h3>
+			<p className="rewind-flow__info">
+				{ translate(
+					'{{strong}}%(restoreTimestamp)s{{/strong}} is the selected point for your restore. ',
+					{
+						args: {
+							restoreTimestamp,
+						},
+						components: {
+							strong: <strong />,
+						},
+					}
+				) }
+			</p>
+			<h4 className="rewind-flow__cta">{ translate( 'Choose the items you wish to restore:' ) }</h4>
+			<RewindConfigEditor currentConfig={ rewindConfig } onConfigChange={ setRewindConfig } />
+			<RewindFlowNotice
+				gridicon="notice"
+				title={ translate( 'This will override and remove all content after this point' ) }
+				type={ RewindFlowNoticeLevel.WARNING }
+			/>
+			<Button
+				className="rewind-flow__primary-button"
+				primary
+				onClick={ onConfirm }
+				disabled={ Object.values( rewindConfig ).every( setting => ! setting ) }
+			>
+				{ translate( 'Confirm restore' ) }
+			</Button>
+		</>
+	);
+
+	const renderInProgress = () => (
+		<>
+			<div className="rewind-flow__header">
+				<Gridicon icon="history" size={ 48 } />
+			</div>
+			<h3 className="rewind-flow__title">{ translate( 'Currently restoring your site' ) }</h3>
+			<ProgressBar percent={ inProgressRewindPercentComplete } />
+			<p className="rewind-flow__info">
+				{ translate(
+					'We are restoring your site back to {{strong}}%(restoreTimestamp)s{{/strong}}.',
+					{
+						args: {
+							restoreTimestamp,
+						},
+						components: {
+							strong: <strong />,
+						},
+					}
+				) }
+			</p>
+			<CheckYourEmail
+				message={ translate(
+					"Don't want to wait? For your convenience, we'll email you when your site has been fully restored."
+				) }
+			/>
+		</>
+	);
+
+	const renderFinished = () => (
+		<>
+			<div className="rewind-flow__header">
+				<img
+					src="/calypso/images/illustrations/jetpack-restore-success.svg"
+					alt="jetpack cloud restore success"
+				/>
+			</div>
+			<h3 className="rewind-flow__title">
+				{ translate( 'Your site has been successfully restored.' ) }
+			</h3>
+			<p className="rewind-flow__info">
+				{ translate(
+					'All of your selected files are now restored back to {{strong}}%(restoreTimestamp)s{{/strong}}.',
+					{
+						args: {
+							restoreTimestamp,
+						},
+						components: {
+							strong: <strong />,
+						},
+					}
+				) }
+			</p>
+			<Button primary href={ siteUrl } className="rewind-flow__primary-button">
+				{ translate( 'View your website' ) }
+			</Button>
+		</>
+	);
+
+	const renderError = () => (
+		<>
+			<div className="rewind-flow__header">
+				<img
+					src="/calypso/images/illustrations/jetpack-cloud-download-failure.svg"
+					alt="jetpack cloud download error"
+				/>
+			</div>
+			<h3 className="rewind-flow__title">
+				{ translate( 'An error occurred while restoring your site' ) }
+			</h3>
+		</>
```

Comment:
Lets ask them to contact support here. 
On the scan section we include a button to the contact form see
https://github.com/Automattic/wp-calypso/blob/master/client/landing/jetpack-cloud/sections/scan/main.jsx#L89